### PR TITLE
Add Release Notes

### PR DESCRIPTION
**Pull request by shanzzam**
Adding release notes for previous and current release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +142,8 @@ yarn build
 - [x] Improve UX
 - [x] Implement SEP-31 support
 - [ ] Forward sep24.stellar.org traffic to demo-wallet.stellar.org
-- [ ] Deprecate older clients to use this tool instead (sep31-demo-client.netlify.app,sep24.stellar.org, sep6.stellar.org)
+- [ ] Deprecate older clients to use this tool instead
+      (sep31-demo-client.netlify.app,sep24.stellar.org, sep6.stellar.org)
```
Comment (marcelosalloum) [inline README.md:146]: nit: add missing space

```suggestion
      (sep31-demo-client.netlify.app, sep24.stellar.org, sep6.stellar.org)
```

File: README.md [hunk 2]
```diff
@@ -107,6 +108,31 @@ yarn build
...
+### v1.2
+
+- SEP-06 now supported
+- SEP-08 now supported
+- Fixed a [bug](https://github.com/stellar/stellar-demo-wallet/issues/188) when
+  overriding home domain
+- Fixed an [issue](https://github.com/stellar/stellar-demo-wallet/issues/196)
+  where balance amounts were being overwritten
+- Sending to Muxed Accounts is now supported
+
```
Comment (marcelosalloum) [inline README.md:135]: 💡  Releases are usually in descending order, we should have `1.2` at the top followed by `1.1` and finally `1.0`.

This is how we do it in the [monoerepo](https://github.com/stellar/go/blob/master/clients/horizonclient/CHANGELOG.md) and it's how Github organizes the releases in https://github.com/stellar/go/releases

**Review by marcelosalloum — COMMENTED** [review]
Added a couple of suggestions below ⬇️

File: README.md (hunk 2 above)
Comment (shanzzam) [reply README.md]: Good point. Updating the file with most recent release notes first.

**Review by marcelosalloum — APPROVED** [review]
LGTM!

**Pull request merged**